src/components/VideoCapture/__tests__/index.test.tsx

```diff
@@ -0,0 +1,170 @@
+}
+
+const assertInactiveError = (wrapper: ReactWrapper, forceRedo: boolean) => {
+  expect(wrapper.find('#record-video').text()).toEqual('Start')
+  expect(wrapper.find('Timeout').exists()).toBeFalsy()
+
+  const error = wrapper.find('CameraError Error')
+  expect(error.exists()).toBeTruthy()
+
+  if (forceRedo) {
+    expect(wrapper.find('#record-video').prop('disabled')).toBeTruthy()
+    expect(wrapper.find('FallbackButton').text()).toEqual(
+      'selfie_capture.alert.timeout.detail'
+    )
+  } else {
+    expect(error.find('.title').text()).toEqual(
+      'selfie_capture.alert.camera_inactive.title'
+    )
+  }
+}
+
+const MockedVideoLayer: FunctionComponent<VideoOverlayProps> = ({
+  disableInteraction,
+  isRecording,
+  onStart,
+  onStop,
+}) => (
+  <button
+    id="record-video"
+    disabled={disableInteraction}
+    onClick={isRecording ? onStop : onStart}
+  >
+    {isRecording ? 'Stop' : 'Start'}
+  </button>
+)
+
+const defaultProps: VideoCaptureProps = {
+  inactiveError: { name: 'CAMERA_INACTIVE' },
+  onRecordingStart: jest.fn(),
+  onRedo: jest.fn(),
+  onVideoCapture: jest.fn(),
+  renderFallback: jest.fn(),
+  renderVideoOverlay: (props) => <MockedVideoLayer {...props} />, // eslint-disable-line react/display-name
+  trackScreen: jest.fn(),
+}
+
+describe('VideoCapture', () => {
+  beforeAll(() => {
+    jest.useFakeTimers()
+  })
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('renders without crashing', () => {
+    const wrapper = shallow(<VideoCapture {...defaultProps} />)
+    expect(wrapper.exists()).toBeTruthy()
+  })
+
+  describe('when mounted', () => {
+    let wrapper: ReactWrapper
+    let camera: ReactWrapper<CameraProps & WithTrackingProps>
+
+    beforeEach(() => {
+      wrapper = mount(
+        <MockedReduxProvider>
+          <MockedLocalised>
+            <VideoCapture {...defaultProps} title="Fake title" />
+          </MockedLocalised>
+        </MockedReduxProvider>
+      )
+
+      camera = wrapper.find<CameraProps & WithTrackingProps>(Camera)
+    })
+
+    it('renders Camera correctly', () => {
+      expect(wrapper.exists()).toBeTruthy()
+      expect(camera.exists()).toBeTruthy()
+
+      const {
+        buttonType,
+        isButtonDisabled,
+        renderError,
+        renderFallback,
+      } = camera.props()
+      expect(buttonType).toEqual('video')
+      expect(isButtonDisabled).toBeFalsy()
+      expect(renderError).toBeFalsy()
+
+      renderFallback('fake_fallback_reason')
+      expect(defaultProps.renderFallback).toHaveBeenCalledWith(
+        'fake_fallback_reason'
+      )
+
+      expect(wrapper.find('PageTitle').exists()).toBeTruthy()
+      expect(wrapper.find('PageTitle').text()).toEqual('Fake title')
+    })
+
+    it('renders inactive timeout correctly', () => assertTimeout(wrapper, 12))
+
+    describe('when inactive timed out', () => {
+      beforeEach(() => {
+        jest.runTimersToTime(12_000) // 12 seconds - default value
```

Comment:
Isn't `runTimersToTime` renamed to `advanceTimersByTime` [now](https://jestjs.io/docs/timer-mocks#advance-timers-by-time)? Though not really an issue if it still works I guess.

Reply:
Thanks for pointing this out :pray: I guess they haven't completely deprecated the former one yet. But it's worth refactoring.